roles/ceph-keepalived/tasks/pre_requisite.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: install keepalived
+  package:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - keepalived
+  when:
+    - ansible_os_family == 'RedHat'
+
+- name: set_fact vip to vrrp_instance
+  set_fact:
+      vrrp_instances: "{{ vrrp_instances | default([]) | union([{ 'name': 'VI_' + index|string , 'vip': item, 'master': groups[keepalived_group_name][index] }]) }}"
+  loop: "{{ virtual_ips | flatten(levels=1) }}"
+  loop_control:
+      index_var: index
+
+- name: "generate keepalived: configuration file: keepalived.conf"
+  action: config_template
+  args:
+    config_type: ini
+    src: keepalived.conf.j2
+    dest: /etc/keepalived/keepalived.conf
+    owner: "root"
+    group: "root"
+    mode: "0644"
+
+- name: "set master for vrrp_instance: configuration file: keepalived.conf"
```

Comment:
why this isn't part of the keepalived template ?

Reply:
@dsavineau because for now, I don't want to make the template file looks too complex by checking the MASTER/BACKUP state, but I shall try to put this part into the template file in the next version.